src/pages/[platform]/build-a-backend/data/customize-authz/grant-lambda-function-access-to-api/index.mdx

```diff
@@ -140,32 +136,43 @@ export const handler = async (event) => {
 }
 ```
 
-The `amplifyconfiguration.json` file is not automatically accessible to your function. To include it with your function, you must first do a deployment of your backend using either `npx amplify sandbox` or Amplify hosting. Then copy the `amplifyconfiguration.json` file that is placed in your project root into your function directory. Now you can import the file into your function as shown above.
+Use the command below to generate GraphQL client code to call your data backend.
+<Callout info>
+**Note** We are working on bringing the end-to-end typed experience to connect to your data from within function resources without needing this step. If you'd like to provide feedback the experience or have early access, join our [Discord community](https://discord.gg/amplify).
+</Callout>
+
+```sh title="Terminal" showLineNumbers={false}
+npx amplify generate graphql-client-code --out <path-function-handler-dir>/graphql
+```
```

Comment:
I think the callout works, but it might be better to have a newline between the text and the component opening, `<Callout info>`. When you start to nest different components then it will start to bug out.

Also should there be a colon after the "Note"?

```suggestion
<Callout info>

**Note**: We are working on bringing the end-to-end typed experience to connect to your data from within function resources without needing this step. If you'd like to provide feedback the experience or have early access, join our [Discord community](https://discord.gg/amplify).
</Callout>

```sh title="Terminal" showLineNumbers={false}
npx amplify generate graphql-client-code --out <path-function-handler-dir>/graphql
```
```